### PR TITLE
Add investigation guide for Amazon Bedrock Rules

### PR DESCRIPTION
**Pull request by shashank-elastic**
# Pull Request

*Issue link(s)*:  A task in issue https://github.com/elastic/ia-trade-team/issues/456#issuecomment-2445705638

## Summary - What I changed

- Added Investigation Guides for Amazon Bedrock Guardrails

## How To Test

- Unit test should pass 

## Checklist

- [x] Added a label for the type of pr: `bug`, `enhancement`, `schema`, `Rule: New`, `Rule: Deprecation`, `Rule: Tuning`, `Hunt: New`, or `Hunt: Tuning` so guidelines can be generated
- [ ] Added the `meta:rapid-merge` label if planning to merge within 24 hours
- [ ] Secret and sensitive material has been managed correctly
- [ ] Automated testing was updated or added to match the most common scenarios
- [ ] Documentation and comments were added for features that require explanation

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/integrations/aws_bedrock/aws_bedrock_multiple_validation_exception_errors_by_single_user.toml [hunk 1]
```diff
@@ -29,6 +29,39 @@ references = [
...
+
+Bedrock offers a variety of pretrained models from Amazon (such as the Titan series), as well as models from providers like Anthropic, Meta, Cohere, and AI21 Labs.
+
+#### Possible investigation steps
+
+- Identify the user account that caused validation errors in accessing the Amazon Bedrock models.
+- Investigate other alerts associated with the user account during the past 48 hours.
+- Consider the time of day. If the user is a human (not a program or script), did the activity take place during a normal time of day?
+- Examine the account's attempts to access Amazon Bedrock models in the last 24 hours.
+- If you suspect the account has been compromised, scope potentially compromised assets by tracking Aamzon Bedrock model access, prompts generated, and responses to the prompts by the account in the last 24 hours.
```
Comment (terrancedejesus) [inline rules/integrations/aws_bedrock/aws_bedrock_multiple_validation_exception_errors_by_single_user.toml:46]:
```suggestion
- If you suspect the account has been compromised, scope potentially compromised assets by tracking Amazon Bedrock model access, prompts generated, and responses to the prompts by the account in the last 24 hours.
```

Can you fix this for the others

File: rules/integrations/aws_bedrock/aws_bedrock_multiple_validation_exception_errors_by_single_user.toml [hunk 2]
```diff
@@ -29,6 +29,39 @@ references = [
...
+
+### False positive analysis
+
+- Verify the user account that that caused validation errors is a legitimate misunderstanding by users on accessing the bedrock models.
+
+### Response and remediation
+
+- Initiate the incident response process based on the outcome of the triage.
```
Comment (terrancedejesus) [inline rules/integrations/aws_bedrock/aws_bedrock_multiple_validation_exception_errors_by_single_user.toml:54]: I'd add "- Review the permission policies attached to the user, group or role behind these requests to ensure they are authorized and expected to access Bedrock"

Something to that degree, across them all.

**Review by terrancedejesus — APPROVED** [review]
Couple comments. LGTM

File: rules/integrations/aws_bedrock/aws_bedrock_multiple_validation_exception_errors_by_single_user.toml (hunk 1 above)
Comment (shashank-elastic) [reply rules/integrations/aws_bedrock/aws_bedrock_multiple_validation_exception_errors_by_single_user.toml]: Done

**Review by Aegrah — APPROVED** [review]

**Pull request merged**